ydb/mvp/oidc_proxy/oidc_session_create.h

```diff
@@ -142,7 +142,7 @@ class THandlerSessionCreate : public NActors::TActorBootstrapped<THandlerSession
         NHttp::THeaders headers(Request->Headers);
         NHttp::TCookies cookies(headers.Get("cookie"));
 
-        if (!code.Empty() && IsStateValid(state, cookies, ctx)) {
+        if (IsStateValid(state, cookies, ctx) && !code.Empty()) {
```

Comment:
Why did you switch the conditions?
If parameter "code" is empty then it doesn't make sense to check parameter "state"

Reply:
I noticed that in the IsStateValid method, the IsAjaxRequest variable is initialized, which can affect the response code.